tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -301,6 +303,25 @@ def f(b1 : bool, b2 : bool):
     assert f(True,False) == f2(True,False)
     assert f(False,True) == f2(False,True)
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Compilation problem. NumPy causing unreadable Windows output see issue #1405")
+def test_conda_flag_disable(language):
+    def one():
+        return True
+    with pytest.warns(None) as record1:
+        epyccel(one, language='c', is_conda_warnings_disabled = True)
+    assert len(record1) == 0 # Equals 0 on every platform
+
+@pytest.mark.skipif(sys.platform == 'win32', reason="Compilation problem. NumPy causing unreadable Windows output see issue #1405")
+def test_conda_flag_verbose(language):
+    def one():
+        return True
+    with pytest.warns(None) as record1:
+        epyccel(one, language='c', is_conda_warnings_detailed = True)
+    if len(record1)>0:
+        warn_message = record1[0].message
+        p = str(warn_message).split(":")[2].strip('\n')
```

Comment:
I think `strip()` would be more future proof than `strip('\n')` This means that all whitespace would be stripped

Reply:
fixed: b56c2091